src/Components/Publishing/Article.tsx

```diff
@@ -222,6 +226,7 @@ const ArticleDiv: StyledFunction<ArticleContainerProps & React.HTMLProps<HTMLDiv
 
 const ArticleContainer = ArticleDiv`
   margin-top: ${props => props.marginTop || "50px"};
+  ${props => props.viewerIsOpen && "position: fixed;"}
```

Comment:
I gave this a try in storybooks and InfiniteScroll in Force but it looks like the scroll position gets reset to the top of the article after exiting. You might be better off targeting `body` + using `overflow` to disable scroll which is how we do it for modals in [Force](https://github.com/artsy/force/blob/master/desktop/components/scrollbar/index.styl#L1-L2).

Reply:
Hm, do you know how to target out to the body without using jquery?